heron/instance/src/java/com/twitter/heron/instance/OutgoingTupleCollection.java

```diff
@@ -51,7 +56,10 @@ public OutgoingTupleCollection(
     this.helper = helper;
     this.systemConfig =
         (SystemConfig) SingletonRegistry.INSTANCE.getSingleton(SystemConfig.HERON_SYSTEM_CONFIG);
+
+    // Read the config values
     this.dataTupleSetCapacity = systemConfig.getInstanceSetDataTupleCapacity();
+    this.maxDataTupleSizeInBytes = systemConfig.getInstanceSetDataTupleSizeBytes();
```

Comment:
If this value does not exist in heron_internal.yaml for example if someone has bundled an older version of heron_internals.yaml with their topology for overriding purposes what happens?